app/pages/pdp_subscription.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
```

Comment:
Remove this file. There is a _pdp_page_ already created. Add elements from all pdp types on that file only.
Also you can add comment specifying if the element is from a subscription, physical or other pdp, in order to keep better organized.